src/Main.java

```diff
@@ -1,5 +1,129 @@
+        System.out.println("За 20 минут машина произвела " + bottle1 + " штук бутылок ");
+        System.out.println("За сутки машина произвела " + bottle2 + " штук бутылок ");
+        System.out.println("За три дня машина произвела " + bottle3 + " штук бутылок ");
+        System.out.println("За месяц машина произвела " + bottle4 + " штук бутылок ");
+        System.out.println("Задание 5 ");
+        byte totalCansOfPaint = 120;
+        byte whitePaintCans = 2;
+        byte brownPaintCans = 4;
+        int oneClass = whitePaintCans + brownPaintCans;
+        int totalClasses = totalCansOfPaint / oneClass;
+        int totalWhitePaintCans = totalClasses * whitePaintCans;
+        int totalBrownPaintCans = totalClasses * brownPaintCans;
+        System.out.println("В школе, где " + totalClasses + " классов, нужно " + totalWhitePaintCans + " банок белой краски и " + totalBrownPaintCans + " банок коричневой краски");
+        System.out.println("Задание 6 ");
+        byte numberOfBananas = 5;
+        byte numberOfIseCream = 2;
+        byte numberOfEgs = 4;
+        short mlMilk = 200;
+        byte hundred = 100;
+        byte gOneBananas = 80;
+        byte in100gMilk = 105;
+        byte gOneIseCream = 100;
+        byte gOneEgs = 70;
+        int weightBananas = numberOfBananas * gOneBananas;
+        int weightIseCream = numberOfIseCream * gOneIseCream;
+        int weightEgs = numberOfEgs * gOneEgs;
+        int weightMilk = (mlMilk / hundred) * in100gMilk;
+        int weightSportBreakfastGr = (weightEgs + weightBananas + weightMilk + weightIseCream);
+        int grPerKg = 1000;
+        float weightSportBreakfastKg = weightSportBreakfastGr / (float) grPerKg;
+        System.out.println("Вес спортзавтрака " + weightSportBreakfastGr + " гр");
+        System.out.println("Вес спортзавтрака " + weightSportBreakfastKg + " кг");
+        System.out.println("Задание 7 ");
+        short dietOne = 250;
+        short dietTwo = 500;
+        byte weightKg = 7;
+        short inOneKgGr = 1000;
+        int weightGr = weightKg * inOneKgGr;
+        int quantityDietOne = weightGr / dietOne;
+        int quantityDietTwo = weightGr / dietTwo;
+        short quantityDiets = 2;
+        int average = (dietOne + dietTwo) / quantityDiets;
+        float dayInAverage = weightGr / (float) average;
+        System.out.println("Если спортсмен будет терять в день по " + dietOne + " гр , то он похудеет через " + quantityDietOne);
+        System.out.println("Если спортсмен будет терять в день по " + dietTwo + " гр , то он похудеет через " + quantityDietTwo);
+        System.out.println("В среднем спортсмену портебуется " + dayInAverage + " , чтобы похудеть ");
+        System.out.println("Задание 8 ");
+        int salaryMasha = 67760;
+        int salaryDenis = 83690;
+        int salaryKristina = 76230;
+        int boost = 10;
+        int percent = 100;
+        byte oneYear = 12;
+        float boostSalary = (float) boost / (float) percent;
```

Comment:
```suggestion
        float boostSalary = boost / (float) percent;
```
достаточно только один операнд привести к float